modules/console/pages/config/security/authorization.adoc

```diff
@@ -1,21 +1,130 @@
+
+In Redpanda, a principal is the authenticated identity (such as a username or OIDC subject) used for access control. For details, see xref:manage:security/authentication.adoc#principals[Users, Principals, and Superusers].
+
+IMPORTANT: For any secret values, xref:console:config/configure-console.adoc[use environment variables] instead of hardcoding them in the configuration file. For example, use `KAFKA_SASL_PASSWORD` for the service account password.
+
+There are two ways to use this service account:
+
+=== Static service account only (no login)
+
+Use this mode to connect Redpanda Console to Redpanda with a static account and *no user login*.
+
+- No login screen.
+- All users share the same access level.
+- Redpanda Console roles and RBAC are not enforced.
+- Useful for internal tools.
+
+.Example
+[source,yaml]
+----
+authentication:
+  basic:
+    enabled: false
+  oidc:
+    enabled: false
+
+kafka:
+  sasl:
+    enabled: true
+    impersonateUser: false
+    username: "console-superuser"
+    password: "secret"
+----
+
+=== Static service account with user login
+
+Use this mode to allow users to log in, while still having all API calls executed under a single service account.
+
+- Users authenticate to the UI using basic or OIDC login.
+- All backend communication to Redpanda APIs uses the service account.
+- Redpanda Console roles (`roleBindings`) determine what users can see or do in the UI.
+- Redpanda authorization is evaluated against the service account.
+
+.Example
+[source,yaml]
+----
+authentication:
+  basic:
+    enabled: true
+
+kafka:
+  sasl:
+    enabled: true
+    impersonateUser: false
+    username: "console-superuser"
+    password: "secret"
+
+authorization:
+  roleBindings:
+    - roleName: admin
+      users:
+        - loginType: basic
+          name: alice
+----
+
+This model is recommended if you want to manage RBAC in Redpanda Console, but still want to use a service account for API calls.
 
 == Roles
 
-Roles defined in Redpanda Console are separate from roles in Redpanda. Roles in Redpanda Console control what a user can see and do within the Console interface and only apply to endpoints where user impersonation is disabled. For example, while an admin user in Redpanda Console can generate debug bundles, the user must also have the corresponding `MANAGE_DEBUG_BUNDLE` ACL in Redpanda to execute the operation.
+Roles in Redpanda Console determine which UI features users can access. These roles are configured in the `roleBindings` stanza of the Redpanda Console configuration file and are evaluated only when impersonation is disabled.
 
-Redpanda Console comes with the following roles:
+.When impersonation is enabled, Redpanda Console uses the identity of the logged-in user to send requests to Redpanda
```

Comment:
Not sure I recognize the user benefit of this collapsible. Is this "nice to know" information that is not critical to user understanding of roles? Found the intro odd because there is no end punctuation, but when you open it up, all the statements have end punctuation but this first one. 
To make this consistent with the next option (when impersonation is disabled), I'd get rid of this and place in bullets.

Reply:
I've rephrased as a question.